### PR TITLE
Added : Landing page

### LINKED ISSUE #4: Add Landing Page

### PR DESCRIPTION
**Pull request by ZuberDunge**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,112 @@
...
+
+.searchbar:focus {
+  border: 2px solid var(--primary-red);
+}
+
+.bat-card {
+  font-size: 14px;
+}
+
+
```
Comment (Ronak-Mutha) [inline src/App.css:37]: You can format this later on using any code formatter, for consistency.

File: src/index.js [hunk 2]
```diff
@@ -3,10 +3,18 @@ import ReactDOM from 'react-dom';
...
 import reportWebVitals from './reportWebVitals';
+import ListContextProvider from './context/data';
+import Home from './home';
+import AuthContextProvider from './context/authContext';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+
+    <ListContextProvider><AuthContextProvider>
```
Comment (Ronak-Mutha) [inline src/index.js:13]: This file can be formatted as well.

**Review by Ronak-Mutha — APPROVED** [review]
Good folder structure and descriptive classnames. Good to merge

File: src/App.css [hunk 3]
```diff
@@ -1,38 +1,112 @@
...
+/* card img */
+
+
+.bat-card-horizontal.bat-card img {
+  width: 180px !important;
```
Comment (Hamdrive) [inline src/App.css:42]: is important necessary to use here?

**Review by Hamdrive — COMMENTED** [review]
Overall good implementation done, apart from other reviewer suggestion just general code formatting needs to be done

File: src/App.css (hunk 3 above)
Comment (ZuberDunge) [reply src/App.css:42]: yeah, due to some reason, but I'll check again.

File: src/index.js (hunk 2 above)
Comment (ZuberDunge) [reply src/index.js:13]: thank you for the feedback, I'll make the changes,

**Pull request merged**